### ISSUE TITLE
Bug: Persistence using @notation is by device, not by account 

### ISSUE THREAD

**Issue by fboerncke** [alexa-games/litexa#130]
# Bug Report

I have multiple Alexa devices in my appartment and found out while testing, that persistence using the @-Notation works by device. This means when I change to another room and try to proceed with a skill this will not work as expected. 

As I can think of use cases for both approaches (save by device vs save by account) this may be an intended behaviour though. In that case I would make this bug report a feature request to allow persistence by Alexa-Account, maybe by having a paramter within litexa.config.js?

## Additional Information

Keep up the good work. You are doing a great job!

**Comment by ayrc** [comment]
Hi Frank,

Litexa indeed does key database entries by deviceId. If you would like to instead key by user, you can follow the steps here in customizing the database key and change it to userId: https://litexa.com/book/backdoor.html#customizing-the-db-key.

Thanks! :]

**Comment by fboerncke** [comment]
Sorry, I was really sure that I carefully read the documentation. 
But I simply overlooked this example.
Thanks, I will try this for sure.

**Comment by ayrc** [comment]
No worries - there's a lot of documentation and so it can be hard to figure out where to search for what you need. Let us know if anything is unclear or if you have suggestions on how to improve the documentation! :)

**Comment by fboerncke** [comment]
I tried this now and it works like described. Thanks a lot. Case closed.

**Comment by ayrc** [comment]
Hey Frank,

We just updated the documentation for overriding the dynamoDB key. The content is exactly the same; we've just moved it to [the Variables and Expressions chapter](https://litexa.com/book/expressions.html#customizing-the-litexa-db-key) as per PR #140. Hopefully this is in a more intuitive location. We've also refreshed the documentation for [What about other Alexa features](https://litexa.com/book/backdoor.html), so feel free to give that a look.
